frontend/controller/actions/identity.js

```diff
@@ -806,6 +643,12 @@ export default (sbp('sbp/selectors/register', {
       throw new Error('Some CIDs could not be deleted')
     }
   },
+  'gi.actions/identity/login': (...params) => {
+    return sbp('okTurtles.eventQueue/queueEvent', 'ACTIONS-LOGIN', ['gi.actions/identity/_private/login', ...params])
+  },
+  'gi.actions/identity/logout': (...params) => {
+    return sbp('okTurtles.eventQueue/queueEvent', 'ACTIONS-LOGIN', ['gi.actions/identity/_private/logout', ...params])
+  },
```

Comment:
Can we replace these with anonymous functions pushed onto the event queue?

Reply:
I think it's better this way for logging, as otherwise you don't know when the anonymous function is executed